std_msgs/Header header

int32[] objects
float32[] box1
float32[] box2
float32[] box3
float32[] box4
int32 num
float32[] confidence
float32[] distances
float32[] x_rel
float32[] y_rel
float32[] yaw_rel

